utils/ratlib.py

```diff
@@ -41,6 +41,37 @@ class Status(Enum):
     """The rescue is open, but is marked inactive"""
 
 
+class Colors:
```

Comment:
Could you turn this into an enum? That would make things a tad more typesafe, you'd just have to grab the color codes with `color.value`.

Reply:
I would agree here, after reviwing enums and coming to terms with it >.>
Colours should be an enum.